main.cpp

```diff
@@ -0,0 +1,2002 @@
+//Author: Dumitru Bogdan-Mihai
+#include <stdio.h>
+#include <string.h>
+#include <iostream>
+#include <stdlib.h>
+#include <fstream>
+#include <windows.h>
+#include <ctime>
+    #define xspaceship 17;
+    #define yspaceship 15;
+    #include "resource.h"
+
+using namespace std;
+
+struct Coordinates{
+    int x, y;
+};
+    struct Coordinates   coord_player_spaceship;
+    struct Coordinates   dim_matrix;
+    struct Coordinates   bonus;
+    struct Coordinates   malus; // malus remove your bonus
+
+struct Present_Minim_Maxim{
+    int present;
+    int minim;
+    int maxim;
+};
+    struct Present_Minim_Maxim     level;
+    struct Present_Minim_Maxim     weapon; // player can fire 1, 2 or 3 rockets simultaneously
+
+enum direction {lleft,rright,up,down};
+    enum direction direction_enemy_move_x = rright;
+    enum direction direction_enemy_move_Y = down;
+
+        int i,j;
+        char m[55][55]; // matricea campului de lupta
+    int nr_lifes=3, nr_cheats, scor=0;
+    int var;    char ch;
+    int nr_Elements_HScore; // nr of saved scores
+    int highScore_int[11]; // keep value of the score
+        char highScore_char[11][55];
+        char newScore_char[55];
+        char cheats[10][15]; // strings of the cheats
+        char cheat_entered[15];
+    // each menu is saven in separate matrix.
+    // Each menu have a number ( nr_menu ) and it keeps track of the current menu.
+        char menu[16][16], menu_settings[16][22], menuHScores[15][18],menu_Instructions[15][15], menu_leved_customize[15][15];
+        int nr_menu, nr_elements_menu[11], highlighted_menu_element;
+    bool quit=false, sound=true, autoplay=false, level_runs=false, menu_is_displayed=true ;
+    bool exists_Bonus = false, exists_Malus = false;
+    bool level_was_changed = false;
+    bool exists_player_rocker[36];
+
+int poz_enemy_spaceships_x[100],poz_enemy_spaceships_y[100],nr_enemy_spaceships;
+int y_limit_rand_move_enemy_spaceships;
+
+void Finding_y_limit_rand_move_enemy_spaceships();
```

Comment:
pick a naming convention and stick to it. either `all_lower_case_with_underscores` or `camelCase`, but don't mix them. it looks weird. btw, C++ developers usually go for the udnerscore version. alo, maybe you could use shorter names?